src/Block/Ssr/RecordList.php

```diff
@@ -118,4 +121,19 @@ private function shouldRedirect(array $result): bool
             || isset($result['resultArticleNumberStatus']) && $result['resultArticleNumberStatus'] === 'resultsFound';
         return count($result['records']) === 1 && $isExactSearch;
     }
+
+    private function isAbsoluteUrl(string $url): bool
+    {
+        $isAbsoluteUrl = preg_match('%^(?:(?:https?|ftp)://)(?:\S+(?::\S*)?@|\d{1,3}(?:\.\d{1,3}){3}|(?:(?:[a-z\d\x{00a1}-\x{ffff}]+-?)*[a-z\d\x{00a1}-\x{ffff}]+)(?:\.(?:[a-z\d\x{00a1}-\x{ffff}]+-?)*[a-z\d\x{00a1}-\x{ffff}]+)*(?:\.[a-z\x{00a1}-\x{ffff}]{2,6}))(?::\d+)?(?:[^\s]*)?$%iu', $url);
```

Comment:
i don't mind copying working examples from the internet but i think we can use this 
`filter_var($URL_T_OCHECK, FILTER_VALIDATE_URL)` - its much more leaner
if returns false, then it's not absolute, therefore we can safely assume, it's relative - at least i don't see any other way FACT-Finder will returns the URL